src/com/projectkorra/projectkorra/configuration/ConfigManager.java

```diff
@@ -100,8 +102,9 @@ public static void configCheck(final ConfigType type) {
 			config.addDefault("Board.MiscSeparator", "  ----------");
 			
 			if (!config.contains("Board.Extras")) {
-				config.addDefault("Board.Extras.RaiseEarthWall", ChatColor.GREEN.getName());
-				config.addDefault("Board.Extras.SurgeWave", ChatColor.AQUA.getName());
+			    config.addDefault("Board.Extras.RaiseEarthWall", Element.EARTH.getColor());

```

Comment:
This seems to cause an issue without `.getColor().toString()`. It causes the object classname to be written to the config